cmd/clusterctl/cmd/config.go

```diff
@@ -21,9 +21,10 @@ import (
 )
 
 var configCmd = &cobra.Command{
-	Use:   "config",
-	Short: "Display clusterctl configuration",
-	Long:  `Display clusterctl configuration.`,
+	Use:     "config",
+	GroupID: groupManagement,
```

Comment:
```suggestion
	GroupID: groupOther,
```
for consistency with kubectl